crates/api/schema.graphql

```diff
@@ -512,6 +512,11 @@ type UserProgramRecordNode implements Node {
   """
   stacksUsed: Int! @juniper(infallible: true, ownership: "owned")
   """
+  A composite score equal to 2 * cpuCycles + 3 * instructions +
+  100 * registersUsed + 200 * stacksUsed
```

Comment:
I don't think we should include the formula here, cause 1. it's SECRET SAUCE and 2. we will 100% forget to update it